scripts/migrate.js

```diff
@@ -94,6 +95,22 @@ const run = async () => {
     rimraf.sync(NAV_DIR);
     rimraf.sync(DICTIONARY_DIR);
 
+    logger.info('Fetching redirects');
+
+    const orderedRedirects = (docs) => {
```

Comment:
[nit[ Can we pull this function outside of the `run` function since it doesn't depend on anything except its argument? This makes just a bit easier to read the `run` script

Reply:
On another note, this doesn't look like it really orders the redirects, but rather sticks them in an object. Did you mean to `sort` these? If not, maybe you could change the name to something that better reflects what this is doing like `groupRedirectsByPage` or something?

Reply:
gotcha. yeah, you're right. it groups them by the Drupal ID, so will rename it.

Reply:
right after I PR'ed this, i realized that i should probably move that `orderedRedirects()` code to `fetch-redirects.js`. wasn't super sure if i should i just include that as part of `fetchAllRedirects()` or as its own function that i call on `fetchAllRedirects()`. i went with the latter and will push up